docs/source/bigchaindb-cli.md

```diff
@@ -20,3 +21,65 @@ There's another command named `bigchaindb-benchmark`. It's used to run benchmark
 $ bigchaindb-benchmark -h
 $ bigchaindb-benchmark load -h
 ```
+
+Note that you can always start `bigchaindb` using a different config file using the `-c` option.
+For more information check the help with `bigchaindb -h`.
+
+
+# Precedence in reading configuration values
+
+Note that there is a precedence in reading configuration values:
+ - (**not yet**) command line;
+ - local config file;
+ - environment vars;
+ - default config file (contained in ``bigchaindb.__init__``).
+
+This means that if the default configuration contains an entry that is:
+
+```
+{...
+    "database": {"host": "localhost",
+                 "port": 28015}
+...}
+```
+
+while your local file `local.json` contains:
+```
+{...
+    "database": {"host": "whatever"}
```

Comment:
Could you give a more concrete example than "whatever"? Maybe show a typical (fake) public DNS name?